docs/basics/networkbehaviour.md

```diff
@@ -109,7 +96,43 @@ If you override the virtual 'OnDestroy' method it's important to alway invoke th
 `NetworkBehaviour` handles other destroy clean up tasks and requires that you invoke the base `OnDestroy` method to operate properly.
 :::
 
-### NetworkBehaviour Pre-Spawn Synchronization
+### Pre-Spawn and MonoBehaviour Methods
+
+Since `NetworkBehaviour` is a derived `MonoBehaviour`, the `NetworkBehaviour.OnNetworkSpawn` method is treated similar to the Awake, Start, FixedUpdate, Update, and LateUpdate `MonoBehaviour` methods when the associated GameObject is active or not active in the hierarchy:
+
+- When active: Awake, Start, FixedUpdate, Update, and LateUpdate are invoked.
+- When not active: Awake, Start, FixedUpdate, Update, and LateUpdate are **not invoked**.
+
+_The unique behavior of OnNetworkSpawn, relative to the previously listed methods, is that it is not invoked until the associated GameObject is active in the hierarchy and its associated NetworkObject is spawned._
+
+If you want to disable a specific `NetworkBehaviour` but still want it to be included in the `NetworkObject` spawn process ( _i.e. so you can still enable it at a later time_), as opposed to disabling the entire `GameObject` you should disable the individual NetworkBehaviour component.
+
+:::caution
+When disabling a NetworkBehaviour component prior to spawning the `NetworkObject` you should not depend upon the `NetworkBehaviour.Start` method for any pre-initialization of properties needed by `NetworkBehaviour.OnNetworkSpawn` as `NetworkBehaviour.Start` will be invoked when the NetworkBehaviour component is enabled where `NetworkBehaviour.OnNetworkSpawn` will be invoked when spawned.
```

Comment:
I made a few tweaks to the first part of this callout. Let me know if you like this better.

> Don't depend on the `NetworkBehaviour.Start` method for pre-initialization properties needed by `NetworkBahviour.OnNetworkSpawn` when disabling a NetworkBehaviour component before spawning the NetworkObject.

The _why_ part is less clear to me. 

> `NetworkBehaviour.Start` is invoked when the NetworkBehaviour component is enabled and `NetworkBehaviour.OnNetworkSpawn` is invoked when spawned.

- When _what_ is spawned? The NetworkObject?
- Why is this the reason I shouldn't depend on the `NetworkBehaviour.Start` method for pre-initialization properties? 
    - Is it because (in this scenario) I'm disabling a NetworkBehaviour component that I still want to include in the NetworkObject spawn process?